config/configgrpc/configgrpc.go

```diff
@@ -245,7 +246,9 @@ func (gcs *GRPCClientSettings) toDialOptions(host component.Host, settings compo
 		if !valid {
 			return nil, fmt.Errorf("invalid balancer_name: %s", gcs.BalancerName)
 		}
-		opts = append(opts, grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingPolicy":"%s"}`, gcs.BalancerName)))
+
+		resolver.SetDefaultScheme("dns")
```

Comment:
Changing a global setting like this has much larger implications than fixing a specific issue. Isn't there more local way of doing it?